src/.nuget/dir.props

```diff
@@ -34,6 +34,7 @@
   <!-- derive an OS Group based on the OS Family -->
   <PropertyGroup>
     <_derivedPackageTargetOSGroup Condition="'$(_derivedPackageTargetOSGroup)' == '' and '$(_runtimeOSFamily)' == 'osx'">OSX</_derivedPackageTargetOSGroup>
+    <_derivedPackageTargetOSGroup Condition="'$(_derivedPackageTargetOSGroup)' == '' and '$(_runtimeOSFamily)' == 'android'">Android</_derivedPackageTargetOSGroup>
```

Comment:
@cydhaselton, `_packageTargetOSGroup` is the property which handles the runtime package import.  If you deleted this line, then I'd expect it to default to importing "Linux" runtime package projects.  That's probably not quite what we want though because it would then be ignoring the "Android" piece of the `SupportedPackageOSGroups`.  Instead, I would leave this code as-is, but add a conditional property definition here https://github.com/dotnet/coreclr/blob/8e986affba47efc1ce85ab0bb9428a6c28dae279/src/.nuget/dir.props#L47, which is something like

```
<_packageTargetOSGroup Condition="'$(_derivedPackageTargetOSGroup)' == 'Android'">Linux</_packageTargetOSGroup>
```

and maybe add a comment explaining that Android is using the same runtime pkgproj's as Linux.
```